Client/Frontend/Browser/Tab.swift

```diff
@@ -146,6 +146,45 @@ class Tab: NSObject {
         }
         return self.url
     }
+    
+    var isFxHomeTab: Bool {
+        if let numberOfUrls = self.sessionData?.urls.count,
```

Comment:
This is always going to be false for tabs that have been loaded and not restored. This might cause bugs when you haven't used the app but it hasn't been evicted from memory yet. Should we handle that case?

Reply:
I think you might have to ELI5 🙊

Reply:
@nishant2718 Sorry 😅. When a tab loads the the first time sessionData gets nil'd out https://github.com/mozilla-mobile/firefox-ios/blob/318b942069c61006a621878687a927facfb3560d/Client/Frontend/Browser/Tab.swift#L328.